sky/cli.py

```diff
@@ -1227,11 +1227,15 @@ def launch(
             clusters=[handle.get_cluster_name()])
         # job_id will be None if no job was submitted (e.g. no entrypoint
         # provided)
+        ret_code = 0
```

Comment:
minor: How about keep the variable naming the same as other places, i.e. `returncode`?